pkg/controller/machineset/controller.go

```diff
@@ -333,48 +342,53 @@ func (r *ReconcileMachineSet) adoptOrphan(machineSet *clusterv1alpha1.MachineSet
 	newRef := *metav1.NewControllerRef(machineSet, controllerKind)
 	ownerRefs = append(ownerRefs, newRef)
 	machine.ObjectMeta.SetOwnerReferences(ownerRefs)
+
 	if err := r.Client.Update(context.Background(), machine); err != nil {
 		klog.Warningf("Failed to update machine owner reference. %v", err)
 		return err
 	}
+
 	return nil
 }
 
 func (r *ReconcileMachineSet) waitForMachineCreation(machineList []*clusterv1alpha1.Machine) error {
 	for _, machine := range machineList {
 		pollErr := util.PollImmediate(stateConfirmationInterval, stateConfirmationTimeout, func() (bool, error) {
-			err := r.Client.Get(context.Background(),
-				client.ObjectKey{Namespace: machine.Namespace, Name: machine.Name},
-				&clusterv1alpha1.Machine{})
-			if err == nil {
+			key := client.ObjectKey{Namespace: machine.Namespace, Name: machine.Name}
+
+			if err := r.Client.Get(context.Background(), key, &clusterv1alpha1.Machine{}); err == nil {
 				return true, nil
-			}
-			klog.Error(err)
```

Comment:
this seems like a minor functional change, BTW.

would it make sense to format this block as:
```
if err := r.Client.Get(context.Background(), key, &clusterv1alpha1.Machine{}); err != nil {
	if apierrors.IsNotFound(err) {
		return false, nil
	}
	klog.Error(err)
	return false, err
}
return true, nil
```
^ untested.